### PR TITLE
feat: [pipe-20916]: add new activities and fix some of them

### PR DESCRIPTION
**Pull request by cjlee01**
add new activities and fix some of them
 also refactor the pull request system comments file

### CODE REVIEW

File: packages/playground/src/components/pull-request/hooks/useEmitCodeCommentStatus.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { useCallback, useEffect } from 'react'
+import { CodeCommentState } from '../interfaces'
+
+export const PR_COMMENT_STATUS_CHANGED_EVENT = 'PR_COMMENT_STATUS_CHANGED_EVENT'
+export const PULL_REQUEST_ALL_COMMENTS_ID = -99999
```
Comment (vardanbansal-harness) [inline packages/playground/src/components/pull-request/hooks/useEmitCodeCommentStatus.ts:5]: Is this supposed to be a minimum integer value? Probably should use `Number.MIN_SAFE_INTEGER` in that case.

File: packages/playground/src/components/pull-request/pull-request-system-label-item.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      header={[
+        {
+          avatar: (
+            <Avatar className="w-6 h-6 rounded-full p-0">
+              <AvatarImage src={avatarUrl} />
+
+              <AvatarFallback>
+                <Text size={1} color="tertiaryBackground">
+                  {/* TODO: fix fallback string */}
+                  {getInitials((payload?.author as PayloadAuthor)?.display_name || '')}
```
Comment (vardanbansal-harness) [inline packages/playground/src/components/pull-request/pull-request-system-label-item.tsx:25]: Isn't `payload?.author` of type `PayloadAuthor`. Any reason we need the type cast here?

File: packages/playground/src/components/pull-request/pull-request-system-label-item.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                </Text>
+              </AvatarFallback>
+            </Avatar>
+          ),
+          name: (payload?.payload?.author as PayloadAuthor)?.display_name,
+          description: (
+            <Text color="tertiaryBackground">
+              {((payload?.payload?.payload as GeneralPayload)?.type as LabelActivity) === 'assign' ? (
+                <>applied </>
```
Comment (vardanbansal-harness) [inline packages/playground/src/components/pull-request/pull-request-system-label-item.tsx:34]: Intentional space after `applied`?

File: packages/playground/src/components/pull-request/pull-request-system-label-item.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+              ) : (
+                <>
+                  {((payload?.payload?.payload as GeneralPayload)?.type as LabelActivity) === 'reassign'
+                    ? 'reassigned '
```
Comment (vardanbansal-harness) [inline packages/playground/src/components/pull-request/pull-request-system-label-item.tsx:38]: Same here

Comment (cjlee01) [reply packages/playground/src/components/pull-request/pull-request-system-label-item.tsx:38]: fixed

File: packages/playground/src/components/pull-request/pull-request-system-label-item.tsx (hunk 3 above)
Comment (cjlee01) [reply packages/playground/src/components/pull-request/pull-request-system-label-item.tsx:34]: fixed

File: packages/playground/src/components/pull-request/hooks/useEmitCodeCommentStatus.ts (hunk 1 above)
Comment (cjlee01) [reply packages/playground/src/components/pull-request/hooks/useEmitCodeCommentStatus.ts:5]: fixed

File: packages/playground/src/components/pull-request/pull-request-system-label-item.tsx (hunk 2 above)
Comment (cjlee01) [reply packages/playground/src/components/pull-request/pull-request-system-label-item.tsx]: fixed

**Review by vardanbansal-harness — APPROVED** [review]

**Pull request merged**